linux_os/guide/system/selinux/selinux-booleans/sebool_secure_mode_insmod/bash/shared.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_ol
```

Comment:
I wonder if the file shouldn't be named `ol.sh` instead of `shared.sh`:
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rule-directories
Same for the Ansible remediation.
@jan-cerny could you confirm which would be the best approach, please?

Besides this, I am also thinking if it's not better to include conditionals in the template remediation. I just saw there is already some conditionals for products there.

Reply:
I agree on that the filename ol.sh/yml could be better but I don't think migrating to conditionals in template makes much sense as these remediations are only applicable to this rule.

Reply:
I agree with you about moving the conditionals to template. I didn't notice initially that they are rule specific. Thanks for clarifying.

Reply:
@Xeicker , FYI, I will follow this PR and promptly review it once do remediations specific for Oracle Linux are renamed. Thanks